src/docs/get-started/flutter-for/android-devs.md

```diff
@@ -2213,10 +2213,10 @@ notifications for your app.
 In Flutter, access this functionality using the
 [Firebase Messaging][] plugin.
 For more information on using the Firebase Cloud Messaging API,
-see the [`firebase_messaging`][] plugin documentation.
+see the [firebase_messaging][] plugin documentation.
 
 [DevTools]: /docs/development/tools/devtools
-[Firebase Messaging]: {{site.github}}/flutter/plugins/tree/master/packages/firebase_messaging
+[Firebase Messaging]: {{site.github}}/FirebaseExtended/flutterfire/tree/master/packages/firebase_messaging
```

Comment:
Are you aware that, though you've changed the "Firebase Messaging" link, that the actual link will take them to the "firebase_messaging" link? Just want to make sure this is what you intended...

Reply:
@sfshaza2 Yes, the change is intentional due to the problems mentioned at #3298 I've updated the link for `Firebase Messaging` and `firebase_messaging`. You may verify they are the right links as I've updated them based on my understanding to avoid the 404.